cmp/grid/GridModel.js

```diff
@@ -697,6 +699,17 @@ export class GridModel {
         if (colIds.length) this.agColumnApi.autoSizeColumns(colIds);
     }
 
```

Comment:
A bit unfortunate that we don't have a local implementation model where we can hang implementation state like this.